deploy.sh

```diff
@@ -29,6 +29,6 @@ sam deploy --stack-name "$stack_name" \
   cri:deployment-source=manual \
   cri:stack-type=localdev \
   --parameter-overrides \
-  Environment=localdev \
+  Environment=dev \
```

Comment:
Any reason for this change? The deploy.sh is for manual deployment and it's recommended to use localdev for most cases when running this script locally.

Reply:
deploy.sh is used by Lime to deploy into our aws dev account?

Which then would fail as Environment=localdev is invalid?

Reply:
Ah interesting, thought it was in that list. I'll see if we can add localdev to. Good catch